test/OrchardCore.Tests/Data/ContentItemTests.cs

```diff
@@ -1,12 +1,43 @@
 using System.Text.Json;
 using System.Text.Json.Dynamic;
 using System.Text.Json.Nodes;
+using OrchardCore.ContentFields.Fields;
 using OrchardCore.ContentManagement;
 
 namespace OrchardCore.Tests.Data
 {
     public class ContentItemTests
     {
+        [Fact]
+        public void JsonNode_WhenParseCalled_ConvertShortTimeFormatToTimeField()
```

Comment:
Since we are forcing the Z timezone then we should have a test with a datetime in a different timezone to ensure that it's saving the value in UTC and deserializing it also in UTC.

Reply:
But how to test that ? the `JOptions`  can not load site's time zone settings 🥴

Reply:
I use "2024-5-31 13:05 +08:00" which is my local time Use the local timezone, but go to the server and run the test using the UTC timezone

Reply:
use this code?
```c#
            Assert.Equal("2024-05-31 13:05", timezoneDateTimeFieldTest.Value.Value.ToUniversalTime().ToString("yyyy-MM-dd HH:mm"));

```

Reply:
Hi @sebastienros looks like we can't serialise it to a time format outside our local time zone, that would change the existing logic

Reply:
Maybe I'm missing your point, can you be more specific? I've added a time test for different time zone formats on PR, but while it passes in CI, it doesn't pass the test locally for me 🥴

Reply:
Done~

Reply:
Seems the build still fail

Reply:
How can I do ~🥴

Reply:
Please check timezone unit tests in [Fluid](https://github.com/sebastienros/fluid)

Reply:
https://github.com/sebastienros/fluid/pull/538

Reply:
Sorry, I'm not familiar with this area. I still feel like I can't get started, 
@hishamco  can you help me? Thanks!

Reply:
Hope to fine time and I will take a look